package.json

```diff
@@ -23,5 +23,8 @@
     "lint-staged": "15.2.2",
     "prettier": "3.2.5",
     "turbo": "1.13.3"
+  },
+  "dependencies": {
+    "@visx/heatmap": "^3.3.0"
```

Comment:
Må legges inn i apps/skde i stedet for rett på rota (`yarn workspace skde add ...`)